components/text-utilities/text-utilites.css

```diff
@@ -0,0 +1,13 @@
+.heading{
+
+}
```

Comment:
class is blank if it is of no use remove it

Reply:
Done